Dockerfile.moc-xdmod

```diff
@@ -86,12 +87,15 @@ RUN chmod 777 /usr/bin/xdmod-openstack-hypervisor \
     && chgrp -R 0 /var/lib \
     && chmod -R g+rwX /root \
     && chgrp -R 0 /root \
-    && chmod -R g+rwX /etc/xdmod \
     && chgrp -R 0 /etc/xdmod \
-    && chmod -R g+rwX /etc/httpd \
-    && chgrp -R 0 /etc/httpd 
+    && find /etc/xdmod -type d -exec chmod 764 {} + \
+    && find /etc/xdmod -type f -exec chmod 664 {} + \
+    && chmod -R 664 /etc/httpd \
+    && chgrp -R 0 /etc/httpd \
+    && find /etc/httpd -type d -exec chmod 764 {} + \
+    && find /etc/httpd -type f -exec chmod 664 {} + \
 
-COPY ./assets/allow_insecure_tokens.py /root
+    COPY ./assets/allow_insecure_tokens.py /root
```

Comment:
I don't think you want this indent here (it makes it look like part of the previous `RUN` command, which it is not).